packages/broadcast/component.js

```diff
@@ -33,24 +32,57 @@ export class FabricBroadcast extends LitElement {
       return;
     }
     if (windowExists) {
-      await this.fetchMessage();
-      setInterval(() => this.fetchMessage(), this.interval);
+      await this._fetchMessage();
+      setInterval(() => this._fetchMessage(), this.interval);
     }
   }
 
-  async fetchMessage() {
+  async _fetchMessage() {
     const url = `${this.api}?path=${this.url}`;
     try {
       const res = await (await fetch(url)).json();
       this._messages = res.length ? res : [];
     } catch (err) {
-      console.error(`failed to fetch broadcasts from given url (${url})`)
+      console.error(`failed to fetch broadcasts from given url (${url})`);
     }
   }
 
+  get _broadcasts() {
+    if (!windowExists) return this._messages;
+    const ignoredBroadcasts = JSON.parse(window.localStorage.getItem('ignored-broadcasts')) || [];
+    return this._messages.filter(({ id }) => !ignoredBroadcasts.includes(id));
+  }
+
+  async _del(id) {
+    const el = this.renderRoot.querySelector(`#${id}`);
+    await el.collapse();
+    const ignoredBroadcasts = JSON.parse(window.localStorage.getItem('ignored-broadcasts')) || [];
+    ignoredBroadcasts.push(id.split('-')[1]);
+    window.localStorage.setItem('ignored-broadcasts', JSON.stringify(ignoredBroadcasts));
+  }
+
   render() {
-    for (const { id, message: text } of this._messages) {
-      this._toastContainer.set({ id: `broadcast-${id}`, text, type: 'warning', canclose: true });
-    }
+    return html`
+      <link
+        rel="stylesheet"
+        type="text/css"
+        href="https://assets.finn.no/pkg/@fabric-ds/css/v1/fabric.min.css"
+      />
+      <aside>
+        ${repeat(
+          this._broadcasts,
+          ({ id }) => `broadcast-${id}`,
+          ({ id, message }) => html`<f-toast
+            class="w-full"
+            id="broadcast-${id}"
+            type="warning"
+            text="${message}"
+            canclose
+            @close=${() => this._del(`broadcast-${id}`)}
+          >
+          </f-toast>`,
+        )}
+      </aside>
+    `;
```

Comment:
This is the main change. The broadcast render method now renders its own markup instead of using the toast api. This means that the broadcasts will appear wherever you use this component.